### PR TITLE
add context to ass.txt

### PR DESCRIPTION
**Pull request by DickwardStewards**
added context. **for other people to know what this file is meant to be.**

### CODE REVIEW

**Comment by Elton-1** [comment]
I would like you to add more context about why they should type in that file.

File: ass.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+This text file is meant for ass.
```
Comment (Elton-1) [inline ass.txt:1]: You should make this all capitalized.

**Review by Elton-1 — COMMENTED** [review]
If you do this i will pull

File: ass.txt (hunk 1 above)
Comment (DickwardStewards) [reply ass.txt:1]: Understood

**Comment by DickwardStewards** [comment]
I updated the file as you requested. please review.

**Pull request merged**